solutions/caesar_encryption.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for encrypting using caesar's cypher.
+
+Module contents:
+    - caesar_encryption: generates an encrypted string.
+
```

Comment:
i think here you have to put more explanation about the model and how it works

Reply:
thankyou so much for taking the time to review this Nada, i really appreciate this.
i will add ir right now.